### PR TITLE
Added Tri-Color SSD1681 Driver

### PR DESCRIPTION
**Pull request by makermelissa**
This PR does several things:
 * Adds a copy the existing SSD1681 (monochrome) driver to SSD1681B
 * Updates the existing SSD1681 driver to work as a Tri-Color display driver
 * Updates the examples to use the new drivers
 * Updates most examples to have consistent sets of initializers

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -56,7 +56,7 @@ confidence=
...
+disable=print-statement,parameter-unpacking,unpacking-in-except,old-raise-syntax,backtick,long-suffix,old-ne-operator,old-octal-literal,import-star-module-level,raw-checker-failed,bad-inline-option,locally-disabled,locally-enabled,file-ignored,suppressed-message,useless-suppression,deprecated-pragma,apply-builtin,basestring-builtin,buffer-builtin,cmp-builtin,coerce-builtin,execfile-builtin,file-builtin,long-builtin,raw_input-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,no-absolute-import,old-division,dict-iter-method,dict-view-method,next-method-called,metaclass-assignment,indexing-exception,raising-string,reload-builtin,oct-method,hex-method,nonzero-method,cmp-method,input-builtin,round-builtin,intern-builtin,unichr-builtin,map-builtin-not-iterating,zip-builtin-not-iterating,range-builtin-not-iterating,filter-builtin-not-iterating,using-cmp-argument,eq-without-hash,div-method,idiv-method,rdiv-method,exception-message-attribute,invalid-str-codec,sys-max-int,bad-python3-import,deprecated-string-function,deprecated-str-translate-call,import-error,bad-continuation,duplicate-code
```
Comment (tannewt) [inline .pylintrc:59]: Please don't change the .pylintrc. You can usually do disables where they apply. Changing this file makes it impossible to know that the settings are consistent across repos.

**Review by tannewt — COMMENTED** [review]
Are there really two different chips? One for color and one for black and white? Why not add a class constructor kwarg for color mode instead?

I see you already added a displayio version here: https://github.com/adafruit/Adafruit_CircuitPython_SSD1681 Does it cover both cases?

Are there plans to deprecate this library in favor of the displayio versions?

Thanks for adding support for these new e-inks!

**Comment by ladyada** [comment]
this is for support on raspberry pi specifically - since we have bonnet's that will use this chipset (all Eink displays went thru a revision, very annoying)

**Comment by makermelissa** [comment]
I talked to @ladyada and the decision was to remove the 1681b file. The duplicate-code check appears to be something that was enabled by default on a very recent version of pylint. For the pylinrc change, it basically compares all the files and sees that there are a lot of similarities and fixing it would require rewriting the entire library. I can look at modify each file to see if that fixes the huge pylint error.

Currently Blinka displayio does not support eInks, so this won't be deprecated until it does. :)

File: .pylintrc (hunk 1 above)
Comment (makermelissa) [reply .pylintrc:59]: Unfortunately even though code is rated 10.00/10.00, the duplicate-code check runs regardless of disables in files. So without modifying the .pylintrc file, it doesn't appear that it is possible to disable this check.

**Comment by kattni** [comment]
I will be looking into the Pylint update tomorrow. It is popping up across the board and causing issues.

**Comment by tannewt** [comment]
> Currently Blinka displayio does not support eInks, so this won't be deprecated until it does. :)

Any plans to add it? You'd only need the `displayio.EPaperDisplay` class right?

> I will be looking into the Pylint update tomorrow. It is popping up across the board and causing issues.

Thanks! I imagine a disable at the top of the file should fix it. Overall, it does seem like a reasonable check to have enabled though.

**Comment by makermelissa** [comment]
> Thanks! I imagine a disable at the top of the file should fix it. Overall, it does seem like a reasonable check to have enabled though.

Unfortunately, this is a repo-level check rather than a file-level check and thus a simple disable at the top of any or even all files doesn't seem make a difference.

**Review by ladyada — APPROVED** [review]

**Comment by makermelissa** [comment]
> Any plans to add it? You'd only need the `displayio.EPaperDisplay` class right?

@tannewt, I just realized I never answered your question. Yes, here's the issue for it: https://github.com/adafruit/Adafruit_Blinka_Displayio/issues/3

**Pull request merged**